base/irrationals.jl

```diff
@@ -24,7 +24,7 @@ abstract type AbstractIrrational <: Real end
 Number type representing an exact irrational value denoted by the
 symbol `sym`, such as [`π`](@ref pi), [`ℯ`](@ref) and [`γ`](@ref Base.MathConstants.eulergamma).
 
-See also [`@irrational`], [`AbstractIrrational`](@ref).
+See also [`@irrational`](@ref Base.@irrational), [`AbstractIrrational`](@ref).
```

Comment:
Possibly this was left off last time to avoid this error:
```
[ Info: CrossReferences: building cross-references.
��� Error: no doc found for reference '[`Base.@irrational`](@ref)' in src/base/numbers.md.
��� @ Documenter.CrossReferences /buildworker/worker/doctest_linux64/build/doc/deps/packages/Documenter/HmHje/src/Utilities/Utilities.jl:32
[ Info: CheckDocument: running document checks.
[ Info: Populate: populating indices.
ERROR: LoadError: `makedocs` encountered an error. Terminating build
```
Should this be documented?